tests/common/util.rs

```diff
@@ -361,6 +362,7 @@ impl CmdResult {
         self.no_stderr().stdout_is_bytes(msg)
     }
 
+    // TODO: implement same functionality asserting as String instead
```

Comment:
Okay, truly the last thing I can find and then it's ready to merge in my opinion if the CI passes. Do we still need this? It makes sense to me that the assertion with a fixture always compares bytes.

Reply:
I removed the TODO. However, I've got some ideas to improve `CmdResult` a little bit, but `CmdResult` itself shouldn't be part of this pr.